src/sentry/search/utils.py

```diff
@@ -18,6 +18,15 @@ class InvalidQuery(Exception):
     pass
 
 
+def parse_release(project, value):
+    # TODO(dcramer): add environment support
+    if value == 'latest':
```

Comment:
Yeah, I agree with this. I'd rather reject at creation time rather than query for it's existence here.